sources/packages/backend/apps/api/src/route-controllers/designation-agreement/_tests_/e2e/designation-agreement.institutions.controller.submitDesignationAgreement.e2e-spec.ts

```diff
@@ -27,25 +27,41 @@ describe("DesignationAgreementInstitutionsController(e2e)-submitDesignationAgree
   let app: INestApplication;
   let db: E2EDataSources;
   let collegeF: Institution;
+  let collegeC: Institution;
   let collegeFLocation: InstitutionLocation;
+  let collegeCLocation: InstitutionLocation;
   let testingModule: TestingModule;
 
   beforeAll(async () => {
     const { nestApplication, dataSource, module } =
       await createTestingAppModule();
     app = nestApplication;
     db = createE2EDataSources(dataSource);
-    const { institution } = await getAuthRelatedEntities(
+    let { institution } = await getAuthRelatedEntities(
```

Comment:
Please don't reuse the same variable, create another constant instead.

Reply:
updated